app/views/membership_applications/_reason_waiting.html.haml

```diff
@@ -96,9 +94,10 @@
 
 
   let initialize = (function() {
-    reasons_list.options.add(other_reason_option);  // add the other_reason to the list
+    // reasons_list.options.add(other_reason_option);  // add the other_reason to the list
```

Comment:
Not need since I added the option directly to the options passed to `select_tag`.

Also, this statement was causing a test failure when I used poltergeist as the web driver.  I then tried using Selenium and it worked.  I have seen poltergeist have trouble with some JS code before, and Selenium generally does a better job with tricky JS.  

However, refactoring eventually removed the need to use Selenium.